deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: keycloak-operator
           # Replace this with the built image name
-          image: quay.io/keycloak/keycloak-operator:master
+          image: quay.io/keycloak/keycloak-operator:main
```

Comment:
@vmuzikar it seems we need to create/publish the tag `main` for the operator. When I try to run: podman pull quay.io/keycloak/keycloak-operator:main

I get the following output:
```
Trying to pull quay.io/keycloak/keycloak-operator:main...
Error: initializing source docker://quay.io/keycloak/keycloak-operator:main: reading manifest main in quay.io/keycloak/keycloak-operator: manifest unknown: manifest unknown
```

Reply:
I assumed there will be the `main` tag as soon as a new PR is merged. That is thanks to:
https://github.com/keycloak/keycloak-operator/blob/main/.github/workflows/container.yml#L34